### PR TITLE
Require matplotlib <3.7.0

### PR DESCRIPTION
**Pull request by mwittgen**
Checklist
* [X] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [X] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by mwittgen** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -145,7 +145,7 @@ outputs:
         - iminuit >=2
         - lmfit <1.0.3
         - lsstdesc.coord
-        - matplotlib-base !=3.6.0
+        - matplotlib-base !=3.6.0,<3.7.0
```
Comment (ktlim) [inline recipe/meta.yaml:148]: Are you sure this shouldn't be `<3.6.0`, which would be clearer?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {% set name = "rubin-env" %}
-{% set version = "5.0.0" %}
-{% set build_number = 2 %}
+{% set version = "5.0.1" %}
```
Comment (ktlim) [inline recipe/meta.yaml:2]: In theory, a `<` pin can be a new build, but you need the patch to avoid reusing already-built environments?

**Review by ktlim — APPROVED** [review]

File: recipe/meta.yaml (hunk 2 above)
Comment (mwittgen) [reply recipe/meta.yaml:2]: Not increasing the build number makes sense. Old environments won't build otherwise. Fixing it.

**Pull request merged**